app/components/work_packages/types/pattern_input.rb

```diff
@@ -49,6 +49,7 @@ def suggestions_for_stimulus
       def suggestions_list_component
         @suggestions_list_component ||= Primer::Alpha::ActionList.new(
           role: :list,
+          classes: "mr-3",
```

Comment:
Please do not use the helper classes generated by the system arguments directly. Primer might change that, and it will not work any more. Please use the system arguments instead.

```suggestion
          mr: 3
```